tardis/adapters/sites/htcondor.py

```diff
@@ -70,9 +74,18 @@ def __init__(self, machine_type, site_name):
         submit_jdl = self.configuration.MachineTypeConfiguration[self._machine_type].jdl
         drone_resources = ";".join(
             [f'TardisDrone{resource}={self.machine_meta_data[resource]}' for resource in self.machine_meta_data])
+        submit_resources_args = ''
+        for resource in self.machine_meta_data:
+            try:
+                submit_resources_args += \
+                    f'-a "{htcondor_translate_resources[resource]} = {self.machine_meta_data[resource]}" '
```

Comment:
Units of machine_meta_data and HTCondor are not matching. For example: Memory in machine_meta_data is specified in GB, while memory in HTCondor is expected in MB.